index.html

```diff
@@ -25,21 +25,26 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
   </head>
   <body>
     <div class="container">
       <div class="nav-container">
         <header>
           <nav class="nav-bar wrap-items">
             <a href="#">Gbenga</a>
-            <img src="./images/Union.png" alt="harmburger" id="drop-menu" />
+            <div class="open-menu">
+              <img src="./images/Union.png" alt="hamburger">
+            </div>
             <ul class="nav-links">
-              <li><a href="#">Porfolio</a></li>
-              <li><a href="#">About</a></li>
-              <li><a href="#">Contact</a></li>
-              <li><a href="#"><img src="./images/mail.png" alt="mail"></a></li>
+              <li class="portfolio-toggle"><a href="#works" class="menu">Porfolio</a></li>
+              <li class="about-toggle"><a href="#about" class="menu">About</a></li>
+              <li class="contact-toggle"><a href="#contact" class="menu">Contact</a></li>
+              <li class="home-icon"><img src="./images/home-indicator.png" alt="home icon" class="home-icon"></li>
```

Comment:
- [ ] I see that you are struggling to implement the dividers between links I really feel the frustration but these challenging moments are what make great developers 
- Hint: you can use `border-bottom: 1px solid (and the color )` that way will make your dividers and also you can make them with `<hr>` tag 
so it would be better if you implement them in order to match the Figma design 

- [ ] Your Design right now

![image](https://user-images.githubusercontent.com/61790587/167943823-94d4d7af-f11d-41c3-9f20-f923047d5cb0.png)